app/controllers/katello/api/registry/registry_proxies_controller.rb

```diff
@@ -428,6 +431,14 @@ def save_push_repo_hrefs
       save_pulp_push_distribution_href(pulp_repo_href)
     end
 
+    def enable_container_push_repo_update(root_repo)
+      root_repo.update!(allow_updates: true)
```

Comment:
The `!` means it will throw an error if anything goes wrong (rather than returning false). That's not what we want here-- I'd go the opposite direction instead:

```suggestion
      root_repo.update_columns(allow_updates: true)
```

Using `update_columns` will skip validations and ensure that we aren't foiled by an error.

Reply:
Same as above.